src/main/java/com/autotune/database/service/ExperimentDBService.java

```diff
@@ -135,6 +136,24 @@ public void loadAllExperimentsData() throws Exception {
             }
         }
         experimentInterface.addResultsToLocalStorage(KruizeOperator.autotuneObjectMap, resultDataList);
+
+        // Load Recommendations from DB and save to local
+        List<KruizeRecommendationEntry> recommendationEntries = experimentDAO.loadAllRecommendations();
+        if (null != recommendationEntries && !recommendationEntries.isEmpty()) {
```

Comment:
Can you please add this null check for above 2 block also

Reply:
Added the null checks to the above blocks